src/magicbox/SuggestionsManager.ts

```diff
@@ -55,7 +55,10 @@ export class SuggestionsManager {
   }
 
   private get focusedSuggestion() {
-    return find(this.currentSuggestions, suggestion => suggestion.dom.classList.contains(this.options.selectedClass));
+    return find(this.currentSuggestions, suggestion => {
+      console.log(suggestion);
+      return suggestion.dom.outerHTML.match(/class=([^=]*)([^(\w\-_)])selected("|([^(\w|\-_)]).*")/) !== null;
```

Comment:
Could we use `suggestion.dom.className` instead of matching on outer HTML `class=` ?
Also, could you add a comment that explains what the regex is trying to match on/ an example ?

Because a simpler one might exist...

Reply:
Yeah just to clarify, the problem is that field suggestions don't return the suggestion element directly but instead have it wrapped inside a few divs, so they look like `<div><div><div class="suggestion selected>Suggestion</div></div></div>`.

Right now the problem is we only check the top level div for the selected class but we need to check if any of the divs have the selected class.

Reply:
Could we perform a DOM query to look for a child with the class? e.g.
```
$$(suggestionEl).findClass(this.options.selectedClass)
```